bedrock/releasenotes/tests/test_views.py

```diff
@@ -0,0 +1,49 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from bedrock.releasenotes.views import show_android_sys_req
+
+# def show_android_sys_req(version):
+#     match = re.match(r"\d{1,2}", version)
+#     if match:
+#         num_version = int(match.group(0))
+#         return num_version >= 46
+
+#     return False
```

Comment:
Is keeping this comment intended? Is it likely to get out of date with the source?

Reply:
:facepalm: no i will zap that before merging. Apols!

Reply:
Zapped @alexgibson 🙇